components/CollectionCells/src/MDCCollectionViewCell.m

```diff
@@ -424,16 +424,32 @@ - (void)updateInterfaceForEditing {
 
 #pragma mark - Selecting
 
++ (NSBundle *)bundleWithName:(NSString *)bundleName {
```

Comment:
Ideally we expose this in MDCIcons.  I would add a TODO here noting that.

Consider a bit more context on the method name, perhaps:
- bundleWithIconName:(NSString *)iconName

Naming is hard.